engine/crates/gateway-core/src/cache/partial.rs

```diff
@@ -67,6 +81,31 @@ where
     }
 }
 
+async fn run_update_phase(update_phase: CacheUpdatePhase, cache: Cache) {
+    // Unsure whether I should run these in parallel or series.
```

Comment:
It's still the same CPU usage though, so ideally for CF I guess you should create all future serially and wait on them in parallel? Though for memory it's kind of fucked... So 🤷

Reply:
Yeah, it's the same usage overall, it's the blocking for longer that I'm concerned about.  This approach will at least yield the CPU for a bit while we write to the cache.  But I dunno, hard to know the impact without properly measuring it.

Reply:
ah, I see what you mean, CPU for other requests. Fair point